room_assistance/indico_room_assistance/forms.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from datetime import date, datetime, time, timedelta
+
+from flask import request, session
+from wtforms import SelectField
+from wtforms.fields.html5 import IntegerField
+from wtforms.fields.simple import TextAreaField
+from wtforms.validators import DataRequired, NumberRange, Optional, ValidationError
+
+from indico.modules.events.requests import RequestFormBase
+from indico.modules.rb import Location
+from indico.util.date_time import now_utc
+from indico.web.forms.base import IndicoForm, generated_data
+from indico.web.forms.fields import IndicoDateField, IndicoDateTimeField
+from indico.web.forms.validators import DateTimeRange, Exclusive
+
+from indico_room_assistance import _
+
+
+class RoomAssistanceRequestForm(RequestFormBase):
+    start_dt = IndicoDateTimeField(_('When'), [DataRequired(),
+                                               DateTimeRange(earliest=datetime.combine(now_utc().date(), time()))],
```

Comment:
why not just `datetime.now()`? or even better, get the current localtime in the instance's DEFAULT_TIMEZONE. however, since it's a CERN plugin and the server will always be in the same timezone, `datetime.now()` should be fine.

Reply:
I would pass `now_utc()` since in the `DateTimeRange` the value passed to `earliest` is naively localized to `UTC` anyway. So passing `datetime.now()` would create a small inconsistency.